_pages/team.html

```diff
@@ -23,11 +23,13 @@
             <h5>Carter Fendley</h5>
             <p>Carter is the lead programmer on the team and a team captain. He specializes in robot code and augmented reality interfaces. He will drive our robot during 2017 competition. Check out <a href="http://carterfendley.com">his website</a>!</p>
         </div>
-        <div>
-            <img src="/assets/img/team/erik-boesen.png">
-            <h5>Erik Boesen</h5>
-            <p>Erik is the second-in-command on the programming team, and additionally works on outreach and fundraising as well as managing the team website and leading the development of <a href="https://github.com/frc1418/VictiScout">VictiScout</a>, our team's scouting framework. He also leads the UI team and is the creator of <a href="http://frcdashboard.github.io">FRC Dashboard</a>. Check out his website <a href="http://erikboesen.com">here</a>.</p>
-        </div>
+        <div></div>
```

Comment:
So since there are only three of us doesn't it look better to have the profiles centered. The right-justified sorta makes it look strange.